spec.emu

```diff
@@ -637,11 +637,11 @@ contributors: Mark S. Miller, Richard Gibson
       <ins class="block">
 
         <emu-clause id="sec-arraybuffer.prototype.transfertoimmutable">
-          <h1>ArrayBuffer.prototype.transferToImmutable ( )</h1>
+          <h1>ArrayBuffer.prototype.transferToImmutable ( [ _newLength_ ] )</h1>
           <p>This method performs the following steps when called:</p>
           <emu-alg>
             1. Let _O_ be the *this* value.
-            1. Return ? ArrayBufferCopyAndDetach(_O_, *undefined*, ~immutable~).
+            1. Return ? ArrayBufferCopyAndDetach(_O_, _newLength_, ~immutable~).
```

Comment:
this causes it to be coerced (using ToIndex), which violates our new principles, so it should be validated to be an index first i think? eg:
```suggestion
            1. Let _newIndex_ be ? ToIndex(_newLength_);
            1. If _newIndex_ is not _newLength_, throw a TypeError exception.
            1. Return ? ArrayBufferCopyAndDetach(_O_, _newIndex_, ~immutable~).
```

Reply:
A big argument for including the parameter at all was consistency with existing `transfer` and `transferToFixedLength` methods, so I really don't think its treatment should differ.

Reply:
For things which are close cousins of existing APIs, which this is, we said we were generally going to stick with the behavior from the original API.

Reply:
Fair enough.